tools/cpplint/cpplint.py

```diff
@@ -4852,7 +4852,6 @@ def CheckStyle(filename, clean_lines, linenum, file_extension, nesting_state,
   # if(match(prev, " +for \\(")) complain = 0;
   # if(prevodd && match(prevprev, " +for \\(")) complain = 0;
   scope_or_label_pattern = r'\s*(?:public|private|protected|signals)(?:\s+(?:slots\s*)?)?:\s*\\?$'
-  classinfo = nesting_state.InnermostClass()
```

Comment:
Because of this lgtm issue:

https://lgtm.com/projects/g/nlohmann/json/snapshot/4ce8114e8bcc36495bed4b4505157e9bdf984156/files/tools/cpplint/cpplint.py?sort=name&dir=ASC&mode=heatmap#x24ea1ee68e875673:1

I know it'll be overwritten next time you update if upstream hasn't fixed it.

Reply:
I think we should exclude external code from lgtm.

Reply:
If you can do that, that would of course be preferred.

Reply:
It can be configured with a `.lgtm.yml` file, see https://lgtm.com/help/lgtm/lgtm.yml-configuration-file.

Reply:
Ah, I'll do that then.